### ISSUE TITLE
dgram.js:527 throw new Error('Not running');

### ISSUE THREAD

**Issue by mrusme** [bencevans/sonos-cli#4]
WIth every command I'm getting the following exception at the end of that command:

```
dgram.js:527
    throw new Error('Not running'); // error message from dgram_legacy.js
    ^

Error: Not running
    at Socket._healthCheck (dgram.js:527:11)
    at Socket.send (dgram.js:347:8)
    at /opt/node/lib/node_modules/sonos-cli/node_modules/sonos/lib/sonos.js:1049:19
    at Array.map (native)
    at Timeout.sendDiscover [as _onTimeout] (/opt/node/lib/node_modules/sonos-cli/node_modules/sonos/lib/sonos.js:1048:44)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
```

The command runs, though, so I can play/next/etc.

**Comment by dizzi90** [comment]
*+1*

**Comment by sergey-arhipenko** [comment]
+1

**Comment by clarketm** [comment]
There is a PR in the pipeline with a proposed solution, [clear timeouts on socket close #171](https://github.com/bencevans/node-sonos/pull/171)

**Comment by clarketm** [comment]
@mrusme – PR was merged in the [node-sonos]() commit: [d928dc1](https://github.com/bencevans/node-sonos/commit/d928dc178e040a1e93cb0b5e1a84132f0b9fab00). I presume the fix will be bundled in the next patch release.

**Comment by bencevans** [comment]
[node-sonos](https://github.com/bencevans/node-sonos) has been bumped and sonos-cli updated to `v0.3.1` with the change. Thanks @clarketm.

This should fix this issue. If not please shout!
